apps/mobile/src/screens/connection.tsx

```diff
@@ -24,25 +24,25 @@ const components = {
 }
 
 type RiseAction =
-  | ActionDataState<'navigate', { path: string }>
-  | ActionDataState<'navigate-back'>
-  | ActionDataState<'toast', { title: string; message?: string }>
+  | ActionModelState<'navigate', { path: string }>
+  | ActionModelState<'navigate-back'>
+  | ActionModelState<'toast', { title: string; message?: string }>
```

Comment:
As discussed, we really need to do a follow-up where we rename things that are not "ModelState". Probably only applies to: Event, Action, State (client-side) and Response.